packages/mobile/src/account/Contact.tsx

```diff
@@ -0,0 +1,170 @@
+import Button, { BtnTypes } from '@celo/react-components/components/Button'
```

Comment:
Could you name this ContactSupport or something more descriptive? Could be confusing with the actual contact models